src/iosanita/contenttypes/restapi/services/types/get.py

```diff
@@ -24,6 +24,11 @@ class FieldsetsMismatchError(Exception):
         "utenti",
         "ulteriori_informazioni",
     ],
+    "Documento": [
```

Comment:
Non so se i file (principale e alternativi) li metterei in un fieldest a parte, subito dopo default

Reply:
Li ho messi nel fieldset "Formati".